### DISCUSSION TITLE
OMZ Themes not working on Arch Linux

### DISCUSSION THREAD

**Discussion by pedrohrb7** [ohmyzsh/ohmyzsh#10916] [Help] [answered]
Hi everyone,

I recently installed Arch Linux and I don't know what happening, but OMZ is not working.
Already tried remove and install again, but nothing changed.

I'm using Alacritty (tried terminator too) and installed zsh with pacman

```
zsh --version
zsh 5.8.1 (x86_64-pc-linux-gnu)

omz version
master (8f56a8b)

echo $ZSH
/home/pedro/.oh-my-zsh

echo $ZSH_THEME
agnoster
```

Tks for the attention...

![Screenshot from 2022-05-07 20-16-22](https://user-images.githubusercontent.com/33993048/167275152-eb2e3c37-3637-4563-aea4-c96fcc381d9e.png)

**Comment by mcornella** [comment]
There already was a `.zshrc` file and it was kept (see the output from the installation):
``` 
Looking for an existing zsh config...
Found ~/.zshrc. Keeping
```
This means that the installer was run with `--keep-zshrc` or `KEEP_ZSHRC=yes`. Because of that, the default OMZ loading lines aren't in your zshrc file, so OMZ is not actually loaded.

This is the minimum loading lines to load OMZ that you should fill out and put in your zshrc file:
```sh
export ZSH=<path/to/ohmyzsh>
ZSH_THEME=<theme>
plugins=(<plugins> <to> <load>)
source $ZSH/oh-my-zsh.sh
```

**Comment by gmankab** [comment]
Yesterday I wrote to the discord about this problem https://discord.com/channels/642496866407284746/809850465537884170/986647371075174450


my zsh theme looks like his, but in the .zshrc threre robbyrussell theme, and the `echo $ZSH_THEME` also shows `robbyrussell`:

```
         __                                     __   
  ____  / /_     ____ ___  __  __   ____  _____/ /_  
 / __ \/ __ \   / __ `__ \/ / / /  /_  / / ___/ __ \ 
/ /_/ / / / /  / / / / / / /_/ /    / /_(__  ) / / / 
\____/_/ /_/  /_/ /_/ /_/\__, /    /___/____/_/ /_/  
                        /____/                       ....is now installed!


Before you scream Oh My Zsh! look over the `.zshrc` file to select plugins, themes, and options.

• Follow us on Twitter: @ohmyzsh
• Join our Discord community: Discord server
• Get stickers, t-shirts, coffee mugs and more: Planet Argon Shop

gmn@arch ~ % echo $ZSH
/home/gmn/.oh-my-zsh

gmn@arch ~ % echo $ZSH_THEME
robbyrussell

gmn@arch ~ % omz version
master (8168ec0)

gmn@arch ~ % zsh --version
zsh 5.9 (x86_64-pc-linux-gnu)

gmn@arch ~ % echo "$SHELL"
/usr/bin/zsh

gmn@arch ~ % 
```

i'm pretty sure that the problem is in one of the updates oh my zsh, because this problem is also on the old unupdated arch image on which everything worked before, and after i update arch

**Comment by pedrohrb7** [comment, marked as answer]
Hii... 

I was a feel days looking for something that fix this problem and I finally solved this just deleting the file 'zshprofile' from '/etc/zsh'... 
Maybe you'll find more files in this directory, the only file I kept there was 'zprofile' (you can just move the files to another folder just in case you need restore)
I'm using Mabox OS, and this file doesn't exist... so I removed from ALG and when I opened a new terminal the theme was working fine.

Hope this could help you too...

**Reply by gmankab** [reply]
thank you

i tried this:

```
sudo pacman --noconfirm -Rnc zsh
sudo pacman --noconfirm -Sy zsh git
rm "$HOME/.zshrc"
rm -rf "$HOME/.oh-my-zsh"
sudo rm -rf "/etc/zsh"
sh -c "$(curl -fsSL raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"
```
warning: commands order is very important 

and now oh my zsh working prefect
